features/jsonapi/collections.feature

```diff
@@ -0,0 +1,79 @@
+Feature: JSON API collections support
+  In order to use the JSON API hypermedia format
+  As a client software developer
+  I need to be able to retrieve valid JSON API responses for collection attributes on entities.
+
+  @createSchema
+  @dropSchema
+  Scenario: Correctly serialize a collection
```

Comment:
Your scenario isn't correct. You have to divide it into several scenarios, with one for each operation. Each scenario consist of a list of steps: Givens, Whens, Thens. In your scenario here, there is only a list of Whens, no Thens, it's a little insane.
If you want to know more about this, I invite you to follow this link: http://docs.behat.org/en/latest/user_guide.html
You can also take inspiration from what has been done for Hydra: https://github.com/api-platform/core/tree/master/features/hydra